src/vs-bicep/Bicep.VSLanguageServerClient.IntegrationTests/NuGet.config

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <clear />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```

Comment:
Weren't we only allowed to have a single entry for security?  Maybe I'm wrong.  Not sure how this applies to public repos.

Reply:
Unfortunately Microsoft.Test.Apex.VisualStudio is not available on nuget feed. And Microsoft.NET.Test.Sdk is not available on vs-impl. So I need both. We are not going to ship this assembly, so I guess we should be okay. I can't think of any other alternative.

Reply:
Right, tests, it should be fine.